### PR TITLE
Update stagenet URL for merged mining

### PR DESCRIPTION
**Pull request by hansieodendaal**
## Description
- Updated the monerod_url to one that sees active mining. The previous URI did not reflect the active stagenet block
explorer; it is stationary and only basically advances when someone starts an xmrig miner pointing to it.
- Updated default grpc settings - disabled to enabled - in the config files.

This block has evidence of a Tari merge mined block being submitted to stagenet when using the updated monerod_url (compare `Tx public Key`):
https://community.xmr.to/explorer/stagenet/tx/b1a71f841857fdffb8d23465d56ec80ee7332559980213cdda4c1f6a3c5f200d
```
2020-11-03 11:18:31.546754400 [tari_mm_proxy::xmrig] TRACE Monero block: Block header: Major version: 14
Minor version: 14
Timestamp: 1604395082
Previous id: 0xcba3…649f
Nonce: 36616
Miner tx: Prefix: Version: 2
Unlock time: 700714
Extra field: Subfield: Tx public Key: 4ae7a1feea999e61645e6febf8df89c737f283d18c46f6e163aabd98c931899a
Subfield: Nonce: 0898c2cbd78dc6f544
Subfield: Merge mining: 0, 0x55d3…4f96
RCT signature: Signature: RCT type: Null
Tx fee: 0
```

## Motivation and Context
Monero merged mining could not be demonstrated properly with the old monerod_url  that was used.

## How Has This Been Tested?
Tested with all required components for Tari merge mining with Monero stagenet, using the new URL. Used code changes in #2339 to test the URL.

## Types of changes

* [X] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [X] I'm merging against the `development` branch.
* [X] I ran `cargo-fmt --all` before pushing.
* [X] I ran `cargo test` successfully before submitting my PR.
* [X] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: common/config/presets/tari-sample.toml [hunk 1]
```diff
@@ -22,7 +22,7 @@ grpc_wallet_address = "127.0.0.1:18143"
 transport = "tor"
 
 [merge_mining_proxy.ridcully]
-monerod_url = "http://192.110.160.146:38081"
+monerod_url = "http://18.133.55.120:38081"
```
Comment (StriderDM) [inline common/config/presets/tari-sample.toml:25]: Good find +1

File: common/config/presets/windows.toml [hunk 2]
```diff
@@ -190,13 +190,13 @@ peer_seeds = [
 #public_address = "/ip4/172.2.3.4/tcp/18189"
 
 # Enable the gRPC server for the base node. Set this to true if you want to enable third-party wallet software
-#grpc_enabled = false
+grpc_enabled = false
```
Comment (StriderDM) [inline common/config/presets/windows.toml]: These should be true.

Comment (hansieodendaal) [reply common/config/presets/windows.toml]: Good find -)

**Review by StriderDM — APPROVED** [review]
LGTM

**Review by stringhandler — APPROVED** [review]

**Pull request merged**